addons/base_iban/static/src/components/iban_widget/iban_widget.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { CharField } from "@web/views/fields/char/char_field";
+import { useDebounced } from "@web/core/utils/timing";
+import { useService } from "@web/core/utils/hooks";
+
+const { useState, useEffect } = owl;
+
+export class IbanWidget extends CharField {
+    setup() {
+        super.setup();
+        this.state = useState({
+            isValidIBAN: null,
```

Comment:
isValidIBAN sounds like a boolean, but it can have 3 values?

Reply:
You are right but I'm considering `null` as if the variable isn't set.
When the variable isn't `true` or `false`, the widget doesn't display any icon.
It does only when I set it.
(Go through the rest of the comment to understand the thoughts)